ci/cloudbuild/builds/noex.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+source "$(dirname "$0")/../../lib/init.sh"
+source module ci/cloudbuild/builds/lib/bazel.sh
+
+export CC=gcc
+export CXX=g++
+
+mapfile -t args < <(bazel::common_args)
+args+=("--copt=-fno-exceptions")
+
+# Skip examples that require exceptions for error handling.
+bazel test "${args[@]}" --test_tag_filters=-integration-test ... -- \
```

Comment:
Are you planning to have a CMake build without exceptions?  There are a number of `if (GOOGLE_CLOUD_CPP_ENABLE_CXX_EXCEPTIONS)` statements that would go untested if we move the `noex` build to Bazel.

Reply:
I wasn't, but now that you mention it, we probably should. I'll probably just change this build to use cmake.... coming up.

Reply:
Done. PTAL